bundles/com.espressif.idf.ui/plugin.xml

```diff
@@ -137,6 +142,11 @@
                   icon="icons/ESP-IDF List Tools.png"
                   style="push">
             </command>
+            <command
+                  commandId="com.espressif.idf.ui.command.managetoolsinstallation"
+                  icon="icons/ESP-IDFManageToolsInstalltion.png"
+                  style="push">
```

Comment:
Could we correct the spelling here? IDFManageToolsInstalltion.png to IDFManageToolsInstallation.png

Reply:
Looks like the same icon is added twice- this also exists in the tools folder